.github/workflows/scheduled-snapshots.yml

```diff
@@ -103,19 +103,22 @@ jobs:
         path: osbuild-composer
         fetch-depth: 0
 
-    - name: Update schutzfile and open PR
+    - name: Update schutzfile and open PR in osbuild-composer repository
       env:
         GITHUB_TOKEN: ${{ secrets.SCHUTZBOT_GH_TOKEN }}
+        SUFFIX: ${{ steps.generate_suffix.outputs.suffix }}
       run: |
         pushd osbuild-composer
-        sed -i -E 's/("baseurl": ".*)-[0-9]{8,12}\/?"/\1-${{ steps.generate_suffix.outputs.suffix }}"/g' Schutzfile
+        touch tmp
+        jq --arg SUFFIX "$SUFFIX" '(.[] | select(has("repos")) | .repos[] | .x86_64[], .aarch64[] | .baseurl) |= if . | contains("-fedora-") or contains("-fedora-modular-") then . else sub("[0-9]{8}";$SUFFIX) end' Schutzfile > tmp
```

Comment:
This is still quite crazy, can you comment in the file what it does so our future selves know what it does?

Reply:
I suppose I could run `grep -r "singleton"` in the repos directory and get a list of repository names from that and use it somehow? I'll try something like that.

Reply:
> We could put something in the json itself indicating to skip/update the baseurl in the workflow, or we could require the baseurl to be an rpmrepo one (whitelist rather than blacklist).

That's a great suggestion, thanks @Gundersanne I didn't realize we can put whatever in the `Schutzfile` so I'll just add a boolean to each repo file to indicate whether it can be updated or not which will also greatly simplify the `jq` command.

Reply:
So I've created https://github.com/osbuild/osbuild-composer/pull/2590 which will simplify the `jq` expression to:
```
(.[] | select(has("repos")) | .repos[] | select(.updateable) | .x86_64[], .aarch64[] | .baseurl) |= sub("[0-9]{8}"; "SUFFIX")
```
This should make more sense and also makes it easy to just affect selected repositories.